src/aws/osml/model_runner/common/metrics_utils.py

```diff
@@ -10,5 +10,5 @@ def build_embedded_metrics_config():
     metrics_config = get_config()
     metrics_config.service_name = "OSML"
     metrics_config.log_group_name = "/aws/OSML/MRService"
```

Comment:
Should the log group name be fed from the metrics name space or should they share a common tag or something along those lines?

Reply:
There is a lot we can clean up with these metrics configuration but I restrained myself to only the new feature implementation in this PR.

For example, as far as I can tell the settings for service_name and log_group_name are not really used in our configuration. A few lines down we set the environment to "local" which just means we dump these EMF records out to STDOUT (see: [aws-embedded-metrics-python/.../local_environment.py](https://github.com/awslabs/aws-embedded-metrics-python/blob/master/aws_embedded_metrics/environment/local_environment.py)). It looks like we are really relying on FireLens to send the logs to the /aws/OSML/MRService log group ([mr_dataplane.ts#L367](https://github.com/aws-solutions-library-samples/osml-cdk-constructs/blob/main/lib/osml/model_runner/mr_dataplane.ts#L367)).

The only thing I updated in this PR is the metrics namespace. Given that we've added a tile server application and will likely expand to others in the future it seemed appropriate to add another level. I'll toss a tech debt ticket onto our backlog to go through the rest of this can clean it up.